src/main/java/frc/robot/subsystems/climberSubsystem.java

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix6.hardware.Pigeon2;
+
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+
+public class climberSubsystem extends SubsystemBase {
+  LeftClimberSubsystem m_lcs;
+  RightClimberSubsystem m_rcs;
+  Pigeon2 m_pigeon2;
+
+
+  /** Creates a new climberSubsystem. */
+  public climberSubsystem(LeftClimberSubsystem lcs, RightClimberSubsystem rcs) {
+    m_lcs = lcs;
+    m_rcs = rcs;
+    m_pigeon2 = LimelightNavigation.m_pigeon2;
```

Comment:
the pigeon should be passed into the constructor in `RobotContainer` since multiple classes rely on it. the same is true for the `LimelightNavigation` class.

Reply:
done